src/components/Registration/RegAddEditOrgPerson.vue

```diff
@@ -206,7 +206,7 @@
                 ref="mailingAddressNew"
                 class="mt-4 mb-n6"
                 :editing="true"
-                :schema="enableRules ? PersonAddressSchema : null"
+                :schema="PersonAddressSchema"
```

Comment:
There is other code to enable the rules but it doesn't nothing here, now. (This will be reworked in a future ticket.)